roles/add_livemigration_user/tasks/main.yml

```diff
@@ -24,32 +24,23 @@
       ssh_key_file: .ssh/id_rsa
       ssh_key_passphrase: ""
       force: false
-  - name: create livemigration .ssh directory
+  - name: create livemigration ssh directory
     file:
-      path: "/home/{{ livemigration_user }}/.ssh"
+      path: "/etc/ssh/{{ livemigration_user }}"
```

Comment:
Yocto specific.

Reply:
Why can't we do the same as Yocto for Debian here ?